src/Caliburn.Micro.WinUI3/Caliburn.Micro.WinUI3.csproj

```diff
@@ -0,0 +1,72 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
```

Comment:
could you change this to net8.0-windows10.0.19041    .net 6 is no longer supported